gradle.properties

```diff
@@ -12,6 +12,6 @@ jedisVersion=2.4.1
 junitVersion=4.12
 restAssuredVersion=2.4.1
 toolsVersion=2.0.3-final
-vertxVersion=3.3.3
+vertxVersion=3.4.2

```

Comment:
There seems to be no relevant API changes in other Vertx components (which are used in vertx-redisques) - so vertx-redisques works
- with all Vertx components in version 3.4.2
- and also with the 'mixture', i.e. redis-client in version 3.4.2 and rest of Vertx in versiopn 3.3.3

As we probably will update towards newer Vertx versions anyway I think it's not necessary a.t.m. to _only_ update the redis-client.